### DISCUSSION TITLE
after installing spacy not able to download en language model

### DISCUSSION THREAD

**Discussion by Piyushtyagi000** [explosion/spaCy#13294] [Help: Installation]
Hi,
I have downloaded a .yml file which consist a virtual environment for NLP including some libraries so I have configured in my system and everything is running in D drive of my computer as my environment is activated in D drive after that i have installed spacy in this drive via "conda install conda-forge spacy " this command and it gets installed but when I am running "python -m spacy download en" in this D drive  then i am getting this error     requests.exceptions.SSLError: HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /explosion/spacy-models/master/shortcuts-v2.json (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)'))),      Please help

**Comment by svlandeg** [comment]
Hi!

Are you behind a firewall at work, or such?

In [this thread](https://github.com/explosion/spaCy/issues/7761), Paul explains a workaround to download the model file directly, and install it from file. Let us know if that works for you!
